server/apps/microservices/src/processors/thumbnail.processor.ts

```diff
@@ -49,11 +50,19 @@ export class ThumbnailGeneratorProcessor {
 
     if (asset.type == AssetType.IMAGE) {
       try {
-        await sharp(asset.originalPath, { failOnError: false })
+        await sharp(asset.originalPath, { failOnError: true })
           .resize(1440, 2560, { fit: 'inside' })
           .jpeg()
           .rotate()
-          .toFile(jpegThumbnailPath);
+          .toFile(jpegThumbnailPath)
+          .catch(() => {
+            this.logger.warn(
+              'Failed to generate jpeg thumbnail for asset: ' +
+                asset.id +
+                ' using sharp, failing over to exiftool-vendored',
+            );
+            return exiftool.extractThumbnail(asset.originalPath, jpegThumbnailPath);
```

Comment:
We also need to catch error in here I believe

Reply:
This is awaited, so it'll be caught in the surrounding try/catch.